doc/source/ray-observability/ray-logging.rst

```diff
@@ -2,7 +2,43 @@
 
 Logging
 =======
-This document will explain Ray's logging system and its best practices.
+This document explains Ray's logging system and related best practices.
+
+Internal Ray Logging Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When ``import ray`` is executed, Ray's logger is initialized, generating a sensible configuration given in ``python/ray/_private/log.py``. The default logging level is ``logging.INFO``.
```

Comment:
Shouldn't the default logging level be "WARNING"? "INFO" is normally quite chatty (at least for RLlib).

Reply:
[The default logging level is `INFO`](https://github.com/peytondmurray/ray/blob/76c47941459f68e074e0a01e0d1c34e0d8c5d8ec/python/ray/_private/log.py#L111) - except for RLlib, [which is `WARNING`](https://github.com/peytondmurray/ray/blob/76c47941459f68e074e0a01e0d1c34e0d8c5d8ec/python/ray/_private/log.py#L117) because of the issue raised in [this PR](https://github.com/ray-project/ray/pull/31858). Maybe I should add a note about this to the documentation?

We currently have a test to ensure that the logging level of ray and each subpackage is not inadvertently modified, so that this doesn't happen in the future either.